assets/sass/components/global/_googlesitekit-noscript.scss

```diff
@@ -0,0 +1,50 @@
+/**
+ * Header styles.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-noscript {
+
+	@include googlesitekit-inner-padding;
+	@include shadow;
+
+	background-color: $c-base;
+	margin: 5px 20px 5px 0;
+
+	&__title {
+		margin: 0;
+	}
+}
+
+.no-js {
+
+	#googlesitekit-notice-activated {
+
+		border-left-color: $c-selective-yellow; // noscript renders inside success
```

Comment:
What does this comment mean? I don't quite follow. Do you mean that the noscript tag renders inside a success message? This could use clarification.

Reply:
He's overriding the success/green style of the activation message when JS is disabled to appear as a warning instead since the activation app render target is inside a `.notice.notice-success`. This wasn't part of the IB but I think is a good change to include.

I agree that it isn't obvious by looking at these styles though and it would benefit from a little more clarification.